### PR TITLE
Add initial entities.

### PR DESCRIPTION
**Pull request by Dolinovska**

### CODE REVIEW

File: Culinary/App_Readme/Spring.Rest.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Spring.NET REST Client Framework
```
Comment (bpashkovskyi) [inline Culinary/App_Readme/Spring.Rest.txt]: Remove this file

File: Culinary/App_Readme/Spring.Social.Core.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Spring.NET Social Framework
```
Comment (bpashkovskyi) [inline Culinary/App_Readme/Spring.Social.Core.txt]: Remove this file

File: Culinary/Controllers/HomeController.cs [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+namespace WebApplication3.Controllers
+{
+    public class HomeController : Controller
+    {
+        public ActionResult Index()
+        {
+            return View();
+        }
+
+        public ActionResult Recipes()
```
Comment (bpashkovskyi) [inline Culinary/Controllers/HomeController.cs]: Should be moved to separate controller

File: Culinary/Controllers/HomeController.cs [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        {
+            return View();
+        }
+        public ActionResult Blog()
```
Comment (bpashkovskyi) [inline Culinary/Controllers/HomeController.cs]: You have this action in BlogController

File: Culinary/Models/Category.cs [hunk 5]
```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace Culinary.Models
+{
```
Comment (bpashkovskyi) [inline Culinary/Models/Category.cs]: Move this file to separate class library

File: Culinary/Models/Category.cs [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+    public class Category
+    {
+        public int Id { get; set; }
```
Comment (bpashkovskyi) [inline Culinary/Models/Category.cs]: Id should be [Required]

File: Culinary/Models/IdentityModels.cs [hunk 7]
```diff
@@ -0,0 +1,33 @@
+﻿using System.Data.Entity;
```
Comment (bpashkovskyi) [inline Culinary/Models/IdentityModels.cs]: Move this file to separate class library

File: Culinary/Models/ManageViewModels.cs [hunk 8]
```diff
@@ -0,0 +1,86 @@
+﻿using System.Collections.Generic;
```
Comment (bpashkovskyi) [inline Culinary/Models/ManageViewModels.cs]: Each class class MUST be in separate file

File: Culinary/Models/Recipes.cs [hunk 9]
```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace Culinary.Models
+{
+    public class Recipes
+    {
+        /*post: id, name, abstract, keywords, ingredients, text, image, video, time, portion, price, date, userid, nick*/
```
Comment (bpashkovskyi) [inline Culinary/Models/Recipes.cs]: Id should be [Required]

File: Culinary/Models/Recipes.cs [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+
+        public string Text { get; set; }
+
+        public string Image { get; set; }
+
+        public int Portion { get; set; }
+
+        public DateTime Time { get; set; }
+
+        public int Price { get; set; }
```
Comment (bpashkovskyi) [inline Culinary/Models/Recipes.cs]: How should be display $0.99 price?

File: Culinary/App_Readme/Spring.Rest.txt (hunk 1 above)
Comment (Dolinovska) [inline Culinary/App_Readme/Spring.Rest.txt]: +

File: Culinary/App_Readme/Spring.Social.Core.txt (hunk 2 above)
Comment (Dolinovska) [inline Culinary/App_Readme/Spring.Social.Core.txt]: +

File: Culinary/Controllers/HomeController.cs (hunk 3 above)
Comment (Dolinovska) [inline Culinary/Controllers/HomeController.cs]: +

File: Culinary/Controllers/HomeController.cs (hunk 4 above)
Comment (Dolinovska) [inline Culinary/Controllers/HomeController.cs]: +

File: Culinary/Models/Category.cs (hunk 6 above)
Comment (Dolinovska) [inline Culinary/Models/Category.cs]: +

File: Culinary/Models/IdentityModels.cs (hunk 7 above)
Comment (Dolinovska) [inline Culinary/Models/IdentityModels.cs]: +

File: Culinary/Models/Recipes.cs (hunk 9 above)
Comment (Dolinovska) [inline Culinary/Models/Recipes.cs]: +

File: Culinary/Models/Recipes.cs (hunk 10 above)
Comment (Dolinovska) [inline Culinary/Models/Recipes.cs]: Price should be decimal?

File: Culinary/Models/ManageViewModels.cs (hunk 8 above)
Comment (Dolinovska) [inline Culinary/Models/ManageViewModels.cs]: +

**Pull request closed without merging**